devops/jobs/AppWatcher.groovy

```diff
@@ -61,10 +61,11 @@ class AppWatcher {
 
         // for each application
         extraVars.get("APPS_TO_CONFIG").each { app_name, app_config ->
-            dslFactory.job(extraVars.get("FOLDER_NAME", "DockerCI") + "/" + app_name) {
+            dslFactory.job(extraVars.get("FOLDER_NAME", "DockerCI") + "/application-watchers/" + app_name + "-watcher") {
```

Comment:
This will require an update to the internal documentation to include creation of this folder: https://github.com/edx/jenkins-job-dsl-internal/pull/126.